scripts/compile-yul.ts

```diff
@@ -57,10 +57,13 @@ function preparePaths(path: string, files: string[], outputDirName: string | nul
             return `sources/${val}`;
         })
         .join(' ');
+    const currentWorkingDirectory = process.cwd();
+    console.log(`Yarn project directory: ${currentWorkingDirectory}`);
+
     const outputDir = outputDirName || files[0];
-    let absolutePathSources = `${process.env.ZKSYNC_HOME}/etc/system-contracts/${path}`;
+    let absolutePathSources = `${currentWorkingDirectory}/${path}`;
```

Comment:
I'm re-working this part a little in a separate PR, so that it works well both here and in zksync-2-dev